R/cache.R

```diff
@@ -106,7 +106,7 @@ renv_cache_path_components <- function(path) {
 
 }
 
-renv_cache_synchronize <- function(record, linkable = FALSE) {
+renv_cache_synchronize <- function(record, linkable = FALSE, max_width = NULL) {
```

Comment:
I don't like the idea of adding a print-specific argument to this function. Should it be an option or something else?